ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.html

```diff
@@ -1,178 +1,177 @@
-            [class.sortable]="col.sortable"
-            [attr.data-automation-id]="'auto_id_' + col.key"
-            [class.adf-data-table__header--sorted-asc]="isColumnSorted(col, 'asc')"
-            [class.adf-data-table__header--sorted-desc]="isColumnSorted(col, 'desc')"
-            (click)="onColumnHeaderClick(col)"
-            (keyup.enter)="onColumnHeaderClick(col)"
-            role="button"
-            tabindex="0"
-            title="{{ col.title | translate }}">
-            <span *ngIf="col.srTitle" class="sr-only">{{ col.srTitle | translate }}</span>
-            <span *ngIf="col.title">{{ col.title | translate}}</span>
-        </th>
-        <!-- Actions (right) -->
-        <th *ngIf="actions && actionsPosition === 'right'" class="actions-column">
-            <span class="sr-only">Actions</span>
-        </th>
-    </tr>
-    </thead>
+<thead *ngIf="showHeader">
+<tr>
+    <!-- Actions (left) -->
+    <th *ngIf="actions && actionsPosition === 'left'" class="actions-column">
+        <span class="sr-only">Actions</span>
+    </th>
+    <!-- Columns -->
+    <th *ngIf="multiselect">
+        <md-checkbox [checked]="isSelectAllChecked" (change)="onSelectAllClick($event)"></md-checkbox>
+    </th>
+    <th class="adf-data-table-cell--{{col.type || 'text'}} {{col.cssClass}}"
+        *ngFor="let col of data.getColumns()"
+        [class.sortable]="col.sortable"
+        [attr.data-automation-id]="'auto_id_' + col.key"
+        [class.adf-data-table__header--sorted-asc]="isColumnSorted(col, 'asc')"
+        [class.adf-data-table__header--sorted-desc]="isColumnSorted(col, 'desc')"
+        (click)="onColumnHeaderClick(col)"
+        (keyup.enter)="onColumnHeaderClick(col)"
+        role="button"
+        tabindex="0"
+        title="{{ col.title | translate }}">
+        <span *ngIf="col.srTitle" class="sr-only">{{ col.srTitle | translate }}</span>
+        <span *ngIf="col.title">{{ col.title | translate}}</span>
+    </th>
+    <!-- Actions (right) -->
+    <th *ngIf="actions && actionsPosition === 'right'" class="actions-column">
+        <span class="sr-only">Actions</span>
+    </th>
+</tr>
+</thead>
 
-    <tbody>
-    <ng-container *ngIf="!loading">
-        <tr *ngFor="let row of data.getRows(); let idx = index"
-            tabindex="0"
-            role="button"
-            [class.is-selected]="row.isSelected"
-            [adf-upload]="allowDropFiles && rowAllowsDrop(row)" [adf-upload-data]="row"
-            [ngStyle]="rowStyle"
-            [ngClass]="getRowStyle(row)"
-            (keyup.enter)="onKeyboardNavigate(row, $event)">
+<tbody>
+<ng-container *ngIf="!loading">
+    <tr *ngFor="let row of data.getRows(); let idx = index"
+        tabindex="0"
+        role="button"
+        [class.is-selected]="row.isSelected"
+        [adf-upload]="allowDropFiles && rowAllowsDrop(row)" [adf-upload-data]="row"
+        [ngStyle]="rowStyle"
+        [ngClass]="getRowStyle(row)"
+        (click)="onRowClick(row, $event)"
```

Comment:
This breaks navigation with mouse and default "double-click" mode, removes native implementation for handling Enter key.